Packs/Akamai_SIEM/Integrations/Akamai_SIEM/README.md

```diff
@@ -1,383 +1,152 @@
-<p>
-    Get security event from <a href="https://www.akamai.com/us/en/resources/waf.jsp">Akamai Web Application Firewall (WAF)</a>
-    service.
-    This integration was integrated and tested with <a href="https://developer.akamai.com/api/cloud_security/siem/v1.html"> API version 1.0 of Akamai WAF
-    SIEM</a>
-</p>
-
-<h2>Use Cases</h2>
-<ul>
-    <li>Get security events from Akamai WAF.</li>
-    <li>Analyze security events generated on the Akamai platform and correlate them with security events generated from
-        other sources in Cortex XSOAR</li>
-</ul>
-
-<h2>Detailed Description</h2>
-<p>
-    A WAF (web application firewall) is a filter that protects against HTTP application attacks. It inspects HTTP traffic
-    before
-    it reaches your application and protects your server by filtering out threats that could damage your site
-    functionality or
-    compromise data.
-</p>
-
-<h2>API keys generating steps</h2>
-<ol>
-    <li>Go to `WEB & DATA CENTER SECURITY`>`Security Configuration`>choose you configuration>`Advanced settings`> Enable SIEM integration.</li>
-    <li><a href="https://control.akamai.com/">Open Control panel</a> and login with admin account.</li>
-    <li>Open <code>identity and access management</code> menu.</li>
-    <li>Create user with assign roles <code>Manage SIEM</code> or make sure the admin has rights for manage SIEM.</li>
-    <li>Log in to new account you created in the last step.</li>
-    <li>Open <code>identity and access management</code> menu.</li>
-    <li>Create <code>new api client for me</code></li>
-    <li>Assign API key to the relevant users group, and assign on next page <code>Read/Write</code> access for <code>SIEM</code>.</li>
-    <li>Save configuration and go to API detail you created.</li>
-    <li>Press <code>new credentials</code> and download or copy it.</li>
-    <li>Now use the credentials for configure Akamai WAF in Cortex XSOAR</li>
-</ol>
-
-<h2>Configure Akamai WAF SIEM on Cortex XSOAR</h2>
-<ol>
-    <li>Navigate to&nbsp;<strong>Settings</strong>&nbsp;&gt;&nbsp;<strong>Integrations</strong>
```

Comment:
Go to Settings & Info > Settings > Integrations > Instances.

Reply:
This is the old line, it is now look like this:
"1. Navigate to **Settings** > **Integrations** > **Servers & Services**."